test/support/test_jitter.py

```diff
@@ -25,16 +25,25 @@
 import numpy as np
 import pytest
 
+# This function is used to see if the gpu is available.
+def gpu_device(gpu_number=0):
```

Comment:
Cool. We have some similar functionality in https://github.com/awslabs/gluon-ts/blob/7ef55e4d6824b77703bb15201b180f00617e1abd/src/gluonts/core/component.py#L484 .
Perhaps it makes sense to move this function there and consolidate things.

Reply:
Great, it looks like the `check_gpu_support()` function already checks for GPU support.  I will update to call this instead.  Thanks!